### PR TITLE
Fix issue related to user clicking on back from captcha registration page

### PR DESCRIPTION
**Pull request by rajadilipkolli**

### CODE REVIEW

File: .springBeans [hunk 1]
```diff
@@ -1,15 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (eugenp) [inline .springBeans]: Hey @rajadilipkolli - out of curiosity - why are you removing this particular file?
I'm not against removing it, but it's not related to the topic of this PR.

File: src/main/java/org/baeldung/Application.java [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package org.baeldung.spring;
```
Comment (eugenp) [inline src/main/java/org/baeldung/Application.java]: Why are we moving this class @rajadilipkolli ?
And - more importantly - how is this related to users clicking back on the captcha?

File: .springBeans (hunk 1 above)
Comment (rajadilipkolli) [inline .springBeans]: Hi @eugenp , unfortunately two pull requests got merged together

My first pull request is only to remove html in templates and second one is to refactor code as per http://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#using-boot-locating-the-main-class spring-boot recommendation.

File: src/main/java/org/baeldung/Application.java (hunk 2 above)
Comment (rajadilipkolli) [inline src/main/java/org/baeldung/Application.java]: @eugenp , as told in above review only https://github.com/Baeldung/spring-security-registration/pull/24/commits/0c3b31bce4e9b1dfd67446d7703e49a3ba04287b this matched PR name

Comment (rajadilipkolli) [inline src/main/java/org/baeldung/Application.java]: https://github.com/Baeldung/spring-security-registration/pull/24/commits/92bdc5a6a4ab90d531f4b9d76e79517272dd2a8a this is saperate PR , I wished to open but unfortunately it got merged with  earlier one

File: .springBeans (hunk 1 above)
Comment (eugenp) [inline .springBeans]: Let's separate them - that way I can more easily merge one of them. 
Cheers, 
Eugen.

File: src/main/java/org/baeldung/Application.java (hunk 2 above)
Comment (eugenp) [inline src/main/java/org/baeldung/Application.java]: Sure, no worries, let's separate them and I'll have another look. 
Thanks for the cleanup work, 
Eugen.

File: .springBeans (hunk 1 above)
Comment (rajadilipkolli) [inline .springBeans]: This is separated now

**Comment by rajadilipkolli** [comment]
Now changes are only related to PR description

**Comment by eugenp** [comment]
Hey @rajadilipkolli - one note here. I had a look at the new PR and it again looks like you're changing the package of some classes and also doing some formatting. 
I simply cannot review the PR with formatting changes unfortunately. You'll need to disable whatever automatic formatting you're using and only make the relevant changes. 
Hope that helps. 
Cheers, Eugen.

**Comment by rajadilipkolli** [comment]
@eugenp can you please point out to your eclipse formatter.

**Comment by rajadilipkolli** [comment]
closing in favour of https://github.com/Baeldung/spring-security-registration/pull/27
